recipes-kernel/linux/linux-raspberrypi_6.6.bbappend

```diff
@@ -0,0 +1,12 @@
+FILESEXTRAPATHS:prepend := "${THISDIR}/files:"
+COMPATIBLE_MACHINE:raspberrypi5 = "(raspberrypi5)"
+
+DTB_NAME = "${RPI_SOC_FAMILY}-${MACHINE}-xen"
+RPI_KERNEL_DEVICETREE:append = " broadcom/${DTB_NAME}.dtb"
+
+SRC_URI:append = " \
+    ${@bb.utils.contains("DISTRO_FEATURES", "xen", "file://xen-console.cfg", "", d)} \
+    file://set-cmdline.cfg \
+    file://${DTB_NAME}.dts;subdir=git/arch/${ARCH}/boot/dts/broadcom \
```

Comment:
Have you checked if this file builds? That's because I don't see any changes to the Makefile which will add this to the build

Reply:
I'm checking this every day.